src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -207,6 +207,8 @@ private void LoadCustomProviderFunctions(List<FunctionMetadata> functionMetadata
 
         private void AddMetadataFromCustomProviders(IEnumerable<IFunctionProvider> functionProviders, List<FunctionMetadata> functionMetadataList)
         {
+            _logger.FunctionMetadataProviderParsingFunctions();
```

Comment:
We make the same logging call from `HostFunctionMetadataProvider` as well. But the "Source" column will be able to help us to distinguish it from that one and this one.